README.rst

```diff
@@ -129,6 +129,6 @@ report bugs. For user questions and scientific discussions, please join the
 Interested in Contributing?
 ===========================
 
-Read our `contributing guide <https://github.com/jonescompneurolab/hnn-core/blob/master/CONTRIBUTING.rst>`_.
+Read our `contributing guide <https://jonescompneurolab.github.io/hnn-core/stable/contributing.html>`_.
```

Comment:
I think you should cross-reference instead of use a link: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
That would be much better. I was wondering how to do that...